packages/react/.webpack/webpack.config.js

```diff
@@ -1,6 +1,10 @@
 const path = require('path');
 
 module.exports = {
```

Comment:
I think later, I will improve the webpack config because the react component package could improve some more. Most w.r.t performance.